meta-luneos/recipes-webos/nyx-modules/nyx-modules/0009-Add-wait-touchscreen-conf.patch

```diff
@@ -0,0 +1,22 @@
+From af74a126de7aff1f2f958ad9c3838c0ef7c68a04 Mon Sep 17 00:00:00 2001
+From: Martin Jansa <Martin.Jansa@gmail.com>
+Date: Fri, 7 Dec 2018 16:55:07 +0000
+Subject: [PATCH] files/systemd/nyx.target.d/wait-touchscreen.conf: import from
+ webOS-ports/master
+
+Change-Id: Id8bc7e8217dc92345d8f9a45a26813cef840ecca
+Signed-off-by: Martin Jansa <Martin.Jansa@gmail.com>
+---
```

Comment:
Missing Upstream-Status,

Is this really fixing the build or something in runtime?